src/pages/Schedule.tsx

```diff
@@ -81,15 +103,34 @@ export const Schedule = () => {
 
   useEffect(() => {
     if (baseScheduleData?.schedule && !scheduleError && !scheduleLoading) {
-      const courses = baseScheduleData.schedule.courses;
-      const coursesId = courses.map((course: CourseSection) => {
+      const cleanPayload = JSON.parse(
+        JSON.stringify(baseScheduleData.schedule.courses, (name, val) => {
+          if (name === "__typename") {
+            delete val[name];
+          } else {
+            return val;
+          }
+        })
+      );
+      const coursesId = cleanPayload.map((course: CourseSection) => {
         return { ...course, id: Math.floor(Math.random() * 10000) };
       });
       setScheduleData(coursesId);
       baseScheduleRef.current = coursesId;
     }
   }, [baseScheduleData, scheduleError, scheduleLoading]);
 
+  useEffect(() => {
+    if (!loading) {
+      if (!error && data) {
+        const userData = data.allUsers;
+        setUserData(userData);
+      } else {
+        console.log(error);
+      }
```

Comment:
Same thing here, does this error need to be displayed in a toast to the user or can we safely ignore it?